sdc/rewrites/read_csv_consts.py

```diff
@@ -0,0 +1,161 @@
+# *****************************************************************************
+# Copyright (c) 2020, Intel Corporation All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+#     Redistributions of source code must retain the above copyright notice,
+#     this list of conditions and the following disclaimer.
+#
+#     Redistributions in binary form must reproduce the above copyright notice,
+#     this list of conditions and the following disclaimer in the documentation
+#     and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+# THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+# OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+# WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+# OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+# EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# *****************************************************************************
+
+from numba.rewrites import register_rewrite, Rewrite
+from numba.ir_utils import find_callname, guard, mk_unique_var
+from numba import ir, errors, consts
+
+from sdc.rewrites.ir_utils import remove_unused_recursively
+
+
+def find_build_sequence(func_ir, var):
+    """Reimplemented from numba.ir_utils.find_build_sequence
+    Added 'build_map' to build_ops list.
+    """
+    from numba.ir_utils import (require, get_definition)
+
+    require(isinstance(var, ir.Var))
+    var_def = get_definition(func_ir, var)
+    require(isinstance(var_def, ir.Expr))
+    build_ops = ['build_tuple', 'build_list', 'build_set', 'build_map']
+    require(var_def.op in build_ops)
+    return var_def.items, var_def.op
+
+
+def _new_definition(func_ir, var, value, loc):
```

Comment:
you could use [make_assign](https://github.com/IntelPython/sdc/blob/master/sdc/rewrites/ir_utils.py#L100)

Reply:
I have found make_assign not very obvious. But I have tryed to replace it with make_assign and it works. I am ok to use make_assign.